flake.nix

```diff
@@ -0,0 +1,105 @@
+{
+  description = "Nix flake for zkLLVM";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs";
+    flake-utils.url = "github:numtide/flake-utils";
+    nix-3rdparty = {
+      url = "github:NilFoundation/nix-3rdparty";
+      inputs = {
+        nixpkgs.follows = "nixpkgs";
+        flake-utils.follows = "flake-utils";
+      };
+    };
+    nil-crypto3 = {
+      url = "https://github.com/NilFoundation/crypto3";
+      type = "git";
+      submodules = false;
+      inputs = {
+        nixpkgs.follows = "nixpkgs";
+      };
+    };
+    nil-zkllvm-blueprint = {
+      url = "https://github.com/NilFoundation/zkllvm-blueprint";
+      type = "git";
+      submodules = true;
+      inputs = {
+        nixpkgs.follows = "nixpkgs";
+        nil-crypto3.follows = "nil-crypto3";
+      };
+    };
+  };
+
+  outputs = { self
+    , nixpkgs
+    , flake-utils
+    , nix-3rdparty
+    , nil-crypto3
+    , nil-zkllvm-blueprint
+    }:
+    (flake-utils.lib.eachDefaultSystem (system:
+      let
+        pkgs = import nixpkgs {
+          inherit system;
+          overlays = [ nix-3rdparty.overlays.${system}.default ];
+        };
+        crypto3 = nil-crypto3.packages.${system}.default;
+        blueprint = nil-zkllvm-blueprint.packages.${system}.default;
+
+      in {
+        packages = rec {
+          inherit pkgs crypto3 blueprint;
```

Comment:
This is a copy-paste mistake from our previous discussion.

Reply:
corrected as in the previous PR